project1/application.py

```diff
@@ -98,8 +98,8 @@ def review():
         rating = request.form.get('review_tags')
         review = request.form.get('review_value')
         email = 'satish'
-
-        if (REVIEW.query.filter_by(email = email, isbn = "1234").first() == None) :
+        isbn = "1234"
```

Comment:
You should retrieve from the isbn form field.

Reply:
Updated ma'am